Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -0,0 +1,148 @@
+
+
+## How to add near touch interactions to collidables
+> If you have UnityUI that you would like to make work with touch events, please read the next section
+
+1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+
+1.  Pick a object that you want to raise pointer events. Any ancestor of that object will be able to receive pointer events. For example, you can add a cube to the scene.
+
+1. Make sure there is a collider to that object.
+
+1. Make sure the layer your object is on is a touchable layer. By default, all layers except Ignore Raycast are touchable. You can see which layers are grabbable by inspecting the Touch Layer Masks in your PokePointer prefab.
+
+1. Add a [NearInteractionTouchable](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchable) to your object. 
+
+1. Set "Events to Receive" to Pointer.
+
+1. Click "Fix bounds" and "Fix center"
+
+1. In the scene inspector, you'll see a white outline square and arrow. The arrow pointer to the "front" of the touchable. The collidable will only be touchable from that direction.
+
+1. If you'd like to make your collider touchable from all directions, add a [NearInteractionTouchableVolume](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableVolume) instead.
+
+## How to add near touch interactions to UnityUI
+1. Make sure you have a [PokePointer](Pointers.md#pokepointer) in your MRTK pointer profile. If you are using the default MRTK profile or the default HoloLens 2 profile, you will have this already. If you have a custom profile, open your custom profile and go to look under Input / Pointers  / Pointer Options. Make sure that for the "articulated hand" controller type you have an entry that points to the PokePointer prefab under MRTK.SDK/Features/UX/Prefabs, or create your own prefab and add a [PokePointer](Pointers.md#pokepointer).
+
+1. Add a UnityUI canvas to your scene.
+
+1. Add a [NearInteractionTouchableUnityUI](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionTouchableUnityUI) to your object. 
+
+1. Set "Events to Receive" to Pointer.
+
+
+
+# Examples
+
+## Touch events
+This example creates a cube, makes it touchable, and change color on touch.
+```csharp
+public class MakeTouchableCube : MonoBehaviour
+{
+    void Start()
+    {
+        GameObject cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
```

Comment:
The cube and material creation seems unnecessary. Can we just assume this is on a GameObject already? or alternatively just make this is a static utils function that takes in a GameObject to make touchable, as well remove the monobehavior class

Reply:
> or alternatively just make this is a static utils function that takes in a GameObject to make touchable, as well remove the monobehavior class

I like that idea.

Reply:
What do you think of the new methods, @Troy-Ferrell?